apps/dev-client/android/build.gradle

```diff
@@ -0,0 +1,40 @@
+// Top-level build file where you can add configuration options common to all sub-projects/modules.
+
+buildscript {
+    ext {
+        buildToolsVersion = "29.0.3"
```

Comment:
In the template this had to be updated to `buildToolsVersion = "30.0.3"` for the Android build to work – does it need to be updated here as well?  Or is that only when using the upstream React Native version?